app/views/editions/secondary_nav_tabs/edit/readonly/_guide.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render partial: "editions/secondary_nav_tabs/edit/readonly/components/title", locals: { edition: @resource } %>
+
+<%= render partial: "editions/secondary_nav_tabs/edit/readonly/components/meta_tag_description", locals: { edition: @resource } %>
+
+<section>
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Chapters",
+  heading_level: 3,
+  font_size: "m",
+} %>
+</section>
+
+<section>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Is every chapter part of a step by step?",
+    heading_level: 3,
+    font_size: "s",
+  } %>
```

Comment:
I don't think we should render this content ("Is every chapter part of a step by step?") as a `<h3>` element given that it is (if I understand the page structure correctly) a subsection of ("Chapters"). I think I'd be happy to render this as a `<label>` as in a previous similar instance but I think we should consider all these pages together from an accessibility perspective as a further piece of work.